modules/cta-links-widget/views/widget.html

```diff
@@ -0,0 +1,42 @@
+{% import '@apostrophecms/template:macros/arrows.html' as arrows with context %}
+{% import "link.html" as link %}
+{% set widget = data.widget %}
+
+<div class="widget cta-links-widget">
+  <ol class="cta-links-widget__items">
+    {% for item in data.widget.links %}
+      {% set path = apos.helper.linkPath(item) %}
+      <li class="cta-links-widget__item" id="{{ item._id }}">
+        {{ link.render({
+          label: item.linkText,
+          path: path,
+          target: item.linkTarget,
+          class: 'button cta-links-widget__button'
+        })}}
+        {% if item.helpLabel %}
+          <span class="cta-links-widget__help-label" {% if item.color %} style="color:{{ item.color }}"{% endif %}>
+            {{ item.helpLabel }}
+            {% if item.helpArrow %}
+              <span class="cta-links-widget__arrow cta-links-widget__arrow--{{ item.helpArrow }}">
+                {% if item.helpArrow === 'left' %}
+                  {{ arrows.left() }}
+                {% endif %}
+                {% if item.helpArrow === 'right' %}
+                  {{ arrows.right() }}
+                {% endif %}
+              </span>
+            {% endif %}
+          </span>
+        {% endif %}
+      </li>
+    {% endfor %}
+  </ol>
+</div>
+
+<style>
+  {% for item in data.widget.links %}
+    #{{ item._id }}:hover a {
```

Comment:
This can definitely be done the same way with a class. I tested and don't see a specificity reason for an id.